src/axom/mir/examples/concentric_circles/CMakeLists.txt

```diff
@@ -0,0 +1,19 @@
+# Copyright (c) 2017-2024, Lawrence Livermore National Security, LLC and
+# other Axom Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (BSD-3-Clause)
+
+set( mir_example_dependencies
+     core
+     slic
+     mir
+   )
+
+# Speed up compilation by separating out the execspaces into different runMIR_xxx.cpp files.
+axom_add_executable(
```

Comment:
Should we add a test that runs this example?

Reply:
Added this.